### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [artemsheludko/fresh#7]
The user @carlsonsantana validate your site template "http://artemsheludko.com/fresh/" and found these accessibility errors:
* Page http://artemsheludko.com/fresh/:
  * Issue *6ebbf8d6-8ce7-4fad-b9e6-43553fa39f3d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.46:1. Recommendation: change background to #00875a.
    * **Selector**: `html > body > main > div > div > a`
    * **Context**: `<a href="/fresh/page2/" class="next">Next <i class="fa fa-chevron-ri...</a>`
  * Issue *744c8f42-e709-4b66-a377-c9c0aeed00ce*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.61:1. Recommendation: change background to #767676.
    * **Selector**: `html > body > main > div > div > span`
    * **Context**: `<span class="previous"><i class="fa fa-chevron-left" a...</span>`
  * Issue *3e6d4621-3c53-4f9e-930a-5f93db3dfcf2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(5) > div > p > a`
    * **Context**: `<a class="more" href="/fresh/china-tiger-leaping-gorge/">(Read more...)</a>`
  * Issue *3b0e7425-27b1-40d5-a559-fdb726097f2d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(5) > div > header > div > span`
    * **Context**: `<span>2017, Mar 28&nbsp;&nbsp;&nbsp;&...</span>`
  * Issue *9c6813ee-4e97-4000-b03d-337de34573f0*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.46:1. Recommendation: change text colour to #00aa71.
    * **Selector**: `html > body > main > div > article:nth-child(5) > div > header > h2 > a`
    * **Context**: `<a href="/fresh/china-tiger-leaping-gorge/">China - tiger leaping gorge</a>`
  * Issue *2c9c7890-9b0c-4bf8-a08b-6592b35ffa68*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(4) > div > p > a`
    * **Context**: `<a class="more" href="/fresh/example/">(Read more...)</a>`
  * Issue *b588f6e9-5791-4374-b5d7-b109d9bec9b8*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(4) > div > header > div > span`
    * **Context**: `<span>2017, Mar 28&nbsp;&nbsp;&nbsp;&...</span>`
  * Issue *f606ac41-5fc6-422a-ae77-a090a6119842*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.46:1. Recommendation: change text colour to #00aa71.
    * **Selector**: `html > body > main > div > article:nth-child(4) > div > header > h2 > a`
    * **Context**: `<a href="/fresh/example/">Example</a>`
  * Issue *01222bfa-8c2c-45d1-89c9-fdde218b9c03*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(3) > div > p > a`
    * **Context**: `<a class="more" href="/fresh/the-white-house/">(Read more...)</a>`
  * Issue *76c3454c-2987-4ceb-80a2-58df56a1a7ab*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(3) > div > header > div > span`
    * **Context**: `<span>2017, Mar 28&nbsp;&nbsp;&nbsp;&...</span>`
  * Issue *249dc9af-a6f1-4ab6-bb56-9b6fc33a51a1*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.46:1. Recommendation: change text colour to #00aa71.
    * **Selector**: `html > body > main > div > article:nth-child(3) > div > header > h2 > a`
    * **Context**: `<a href="/fresh/the-white-house/">Wooden house</a>`
  * Issue *537ad408-94d8-4df4-997f-f389f284fbe7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(2) > div > p > a`
    * **Context**: `<a class="more" href="/fresh/zagreb-city/">(Read more...)</a>`
  * Issue *d559bd93-de5c-4c77-99d9-7d592da7ab6d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(2) > div > header > div > span`
    * **Context**: `<span>2017, Mar 28&nbsp;&nbsp;&nbsp;&...</span>`
  * Issue *3218f8cc-8e61-47c4-97bb-5028f7d21d89*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.46:1. Recommendation: change text colour to #00aa71.
    * **Selector**: `html > body > main > div > article:nth-child(2) > div > header > h2 > a`
    * **Context**: `<a href="/fresh/zagreb-city/">Zagreb city</a>`
  * Issue *e77e9469-6f94-48c9-a81d-c4f06c3913d4*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(1) > div > p > a`
    * **Context**: `<a class="more" href="/fresh/an-old-house/">(Read more...)</a>`
  * Issue *c10e6eb4-f250-40dd-8165-f447a85312fb*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #2a882a.
    * **Selector**: `html > body > main > div > article:nth-child(1) > div > header > div > span`
    * **Context**: `<span>2017, Mar 30&nbsp;&nbsp;&nbsp;&...</span>`
  * Issue *7dc7b873-4ba9-4e0d-aa37-5e0a03490a7f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.46:1. Recommendation: change text colour to #00aa71.
    * **Selector**: `html > body > main > div > article:nth-child(1) > div > header > h2 > a`
    * **Context**: `<a href="/fresh/an-old-house/">An old house</a>`
  * Issue *9e023a91-ffee-44b0-ba84-f307f3fee971*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > aside > div > div:nth-child(3) > li:nth-child(4) > a`
    * **Context**: `<a href="https://github.com/{&#10;  " api_url":="" "https:="" api.github.com",="" "baseurl":="" "="" fresh",="" "build_revision":="" "3395a9f3c5988d43fe4143510ec49d405a8d0928",="" "clone_url":="" github.com="" artemsheludko="" fresh.git",="" "contrib...`
  * Issue *2a4483be-0bec-4ee9-a905-06cb12938291*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > aside > div > div:nth-child(3) > li:nth-child(3) > a`
    * **Context**: `<a href="https://plus.google.com/" class="social-links"><i class="social-icons fa fa-go...</a>`
  * Issue *4c976565-68df-4b23-b4be-ea590b882445*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > aside > div > div:nth-child(3) > li:nth-child(2) > a`
    * **Context**: `<a href="https://twitter.com/" class="social-links"><i class="social-icons fa fa-tw...</a>`
  * Issue *5d5f7a65-f667-4566-8cbf-0acfa4d8249d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > aside > div > div:nth-child(3) > li:nth-child(1) > a`
    * **Context**: `<a href="https://www.facebook.com/" class="social-links"><i class="social-icons fa fa-fa...</a>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
